BlogTemplate/Pages/Drafts.cshtml

```diff
@@ -2,20 +2,27 @@
 @model DraftsModel
 @{
     ViewData["Title"] = "Drafts";
+
     <h2>@ViewData["Title"]</h2>
 
-    if (!Model.Posts.Any())
+    if (!Model.DraftSummaries.Any())
     {
         <p>View and edit your saved drafts here.</p>
     }
-
     else
     {
-        foreach (var post in Model.Posts)
+        foreach (var post in Model.DraftSummaries)
         {
             <article>
-                <h2><a asp-page="/edit" asp-route-id="@post.Id" asp-route-slug="@post.Slug">@post.Title</a></h2>
-                <p class="lead">@post.Excerpt</p>
+                <div class="col-md-12">
+                    <h2><a asp-page="/edit" asp-route-id="@post.Id" asp-route-slug="@post.Slug">@post.Title</a></h2>
+                    @if (post.PublishTime != default(DateTimeOffset))
+                    {
+                        <h5 class="date text-muted">Last Published @post.PublishTime.ToString("MMM dd, yyyy")</h5>

```

Comment:
I don't know if `<h5>` is the right semantics.  The published date isn't really a header, it's a bit of metadata.  I'd use something like a span or div and style it appropriately.

Reply:
Even though the published date isn't a header, I used <h5> to make the font style consistent the font style for the headings and other short text (a sans serif font in this case). This will ensure that the published date has a font style consistent with short text if the blog owner chooses to import a different theme from Bootstrap.

Reply:
Even though the published date isn't a header, I used `h5` to make the font style consistent the font style for the headings and other short text (a sans serif font in this case). This will ensure that the published date has a font style consistent with short text if the blog owner chooses to import a different theme from Bootstrap.

Reply:
But you're relying on the default style in the browser, and your semantics are screwed up.  If you want a slightly larget text.  Instead you should use CSS to specify the style and use a semantically accurate tag.

Reply:
Put it this way: which is more important, the date, or the content?